2-mandatory/1-weather-report.js

```diff
@@ -11,10 +11,20 @@
         - Hint: you can call the temperatureService function from your function
 */
 
+cities1 = ['London', 'Paris','Barcelona','Dubai','Mumbai', 'São Paulo','Lagos'];
+
 function getTemperatureReport(cities) {
     // TODO
+    let arr = [];
+    for (let item of cities) {
+        let temp = temperatureService(item);
+        arr.push (`The temperature in ${item} is ${temp} degrees`); 
+    }
+    return arr;
```

Comment:
really nice solution & use of template literals :) could you think of renaming your `arr` variable to something more descriptive? For example `let cityTemperatures`